Code Jam 8 - Qualifier Release Update

janine9vn opened the pull request:
This PR updates the website to include information for the Qualifier Release today. It adds the following:
- Adds a page for the GitHub Bootcamp information. Right now it's fairly basic. It will be more updated in a later PR.
- Adds Tabnine as a sponsor
- Adds a prize section with a quick blurb for each of the sponsors
- Adds information about the qualifier itself and the signup link

janine9vn committed:
Adding GitHub Bootcamp Information
This commit adds some basic information about the github bootcamp.
More information should and will be added later, but this adds some
of the basics and adds it to the sidebar.

janine9vn committed:
Add in information about the Qualifier
This adds in a section about the qualifier and links to it and the
sign-up form.

janine9vn committed:
Add Tabnine as a sponsor
Adds tabnine as a sponsor on the events sidebar.

janine9vn committed:
Adds Prizes section to CJ8 page
This adds a Prizes section to the Code Jam 8 page.
Links, descriptions, and a quick summation of the prizes offered
by each sponsor is included.

The Python Discord prizes should also be included at a later date.

janine9vn committed:
Update qualifier repo and sign-up link

jb3 committed:
Merge branch 'main' into cj8-qualifier-release

ToxicKidz commented on pydis_site/templates/events/pages/code-jams/8/github-bootcamp.html:
```diff
@@ -0,0 +1,40 @@
+    </p>
+    <h3>Session 2: Using git and github in a team environment</h3>
+    <p>
+        This session will run for an hour. It will be focused on using git and HitHub in a team environment.
```
```suggestion
        This session will run for an hour. It will be focused on using git and GitHub in a team environment.
```

mbaruh commented on pydis_site/templates/events/pages/code-jams/8/_index.html:
```diff
@@ -36,10 +38,90 @@ <h3 id="technology"><a href="#technology">Technology</a></h3>
+                        We make it simple to launch in the cloud and scale up as you grow—whether you’re running one virtual machine or ten thousand.
+                    </p>
+                    <p><strong>Prizes</strong><br>
+                    $250 in DigitalOcean credits to a winning team.</p>
```
```suggestion
                    $250 in DigitalOcean credits to the members of a winning team.</p>
```

mbaruh commented on pydis_site/templates/events/pages/code-jams/8/_index.html:
```diff
@@ -36,10 +38,90 @@ <h3 id="technology"><a href="#technology">Technology</a></h3>
+                        We provide tools to explore and familiarize with code bases faster. Our products make it easy for you to take care of quality during all stages of development and spend less time on maintenance tasks.
+                    </p>
+                    <p><strong>Prizes</strong><br>
+                    1-year JetBrain licenses to a winning team.</p>
```
```suggestion
                    1-year JetBrain licenses to the members of a winning team.</p>
```

mbaruh commented on pydis_site/templates/events/pages/code-jams/8/_index.html:
```diff
@@ -36,10 +38,90 @@ <h3 id="technology"><a href="#technology">Technology</a></h3>
+                    Tabnine comes as a plugin and has a free-forever basic plan, so you can get started with it right away!
+                    </p>
+                    <p><strong>Prizes</strong><br>
+                    1-year Pro Licenses to Tabnine to a winning team.</p>
```
```suggestion
                    1-year Pro Licenses to Tabnine to the members of a winning team.</p>
```

janine9vn committed:
Spell GitHub's name correctly
One day I'll spell GitHub correctly and consistently. One day...

janine9vn committed:
Clarify prize details
Clarify that each member of a winning team gets one of the licenses as a
prize. They don't have to Lord of Flies duel it out.

janine9vn committed:
Merge remote-tracking branch 'origin/cj8-qualifier-release' into cj8-qualifier-release
